send-push.js

```diff
@@ -0,0 +1,55 @@
+
+//1.  Uncomment the following with the environment you'd like to use
+//var pusherSetup = require ('./prod-pusher-keys.js');
+var pusherSetup = require ('./dev-pusher-keys.js');
+
+
+
+//2. Fill out the following:
+//to_notify can be "general" or the project ID
+//project_id will show a link to the project if the user opens the notification and if the project is in the mobile-friendly list
+var to_notify = "general"
+var project_id = "21"
+var title = "New Images on Galaxy Zoo"
+var body = "We’re extremely excited to announce the release of a new dataset on Galaxy Zoo."
```

Comment:
Nice. So whenever this file is run the information from these few lines will be pushed to the dev or prod environments, depending on which is defined?